src/player.js

```diff
@@ -1,11 +1,9 @@
 class Player {
-	constructor(name) {
-		this.name = name;
-		this.currentScore = 0;
-		this.totalScore = 0;
-	}
+  constructor(name) {
+    this.name = name;
+    this.totalScore = 0;
+    this.currentScore = 0;
```

Comment:
I think I might be a little confused between `totalScore` and `currentScore` -- I get that currentScore might be the score for the current round...but if so, let's be that specific. `currentRoundScore` perhaps?